internal/pkg/service/stream/storage/level/local/encoding/size/size.go

```diff
@@ -102,7 +102,9 @@ func NewMeterWithBackup(ctx context.Context, clk clock.Clock, logger log.Logger,
 func (m *Meter) Write(p []byte) (int, error) {
 	n, err := m.w.Write(p)
 	m.lock.Lock()
-	m.size += datasize.ByteSize(n)
+	if n > 0 {
+		m.size += datasize.ByteSize(n)
+	}
```

Comment:
Why do we need 1 if instruction ? 45 + 0 = 45, or?

Reply:
```
Write writes len(p) bytes from p to the underlying data stream. It returns the number of bytes written from p (0 <= n <= len(p)) and any error encountered that caused the write to stop early. Write must return a non-nil error if it returns n < len(p). Write must not modify the slice data, even temporarily.
```
Therefore there cannot be negative number of written bytes, only error.

Reply:
https://github.com/keboola/keboola-as-code/pull/2065/commits/a95b10318f90f9d4d3fd22ad5ec7d71d9d458cdb